apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -518,6 +523,18 @@ export class ChangeAction extends BaseRushAction {
     return bumpOptions;
   }
 
+  private async _isEmailRequired(): Promise<boolean>  {
```

Comment:
nit: this function doesn't do anything async, so I'd just return a raw boolean instead of a promise for a boolean.